lib/server/server-google-pubsub.ts

```diff
@@ -251,9 +251,9 @@ export class GooglePubSubTransport extends Server implements CustomTransportStra
         const _topicName = this.topicNamingStrategy.generateTopicName(topicName);
         const topic: GooglePubSubTopic | null = this.googlePubSubClient.getTopic(_topicName);
 
-        return await this.googlePubSubClient
+        return (await this.googlePubSubClient
             .createSubscription(subscriptionName, topic, createOptions)
-            .toPromise();
+            .toPromise()) as GooglePubSubSubscription | null;
```

Comment:
What's the reason for the bulk option here?

Reply:
Convention with in the library.